Kitodo/src/main/java/org/kitodo/services/file/FileService.java

```diff
@@ -241,15 +285,17 @@ public Integer getNumberOfFiles(URI directory) {
      */
     public Integer getNumberOfImageFiles(URI directory) {
         int count = 0;
-        if (isDirectory(directory)) {
-            /*
-             * die Images zählen
-             */
+        if (directory.isAbsolute()) {
+            if (isDirectory(directory)) {
+                count = getSubUris(Helper.imageNameFilter, directory).size();
```

Comment:
I didn't add it, I was not changing logic of the method:
https://github.com/Beacze/kitodo-production/commit/e0b8a5e341e691cb7bb2a39a9f3088ea480bd871